demo/index.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<!--
+    @license
```

Comment:
I double-checked the seed-element, and I can't find any usage of this indentation for the license. Am I looking in the right place?

https://github.com/PolymerLabs/seed-element/blob/0.8-preview/seed-element.html
https://github.com/PolymerLabs/seed-element/blob/0.8-preview/demo/index.html